txnpool/proc/txnpool_worker.go

```diff
@@ -41,14 +41,20 @@ type pendingTx struct {
 	flag    uint8           // For different types of verification
 	retries uint8           // For resend to validator when time out before verified
 	ret     []*tc.TXAttr    // verified results
+	height  uint32
+}
+
+type reValidateTx struct {
+	height uint32 // To be re-verified at the height
```

Comment:
tracking height in each transaction validation is obvious unsound, since when a new block checking request arrive, the old one will be replaced, and the height will change.